Helper/Data.php

```diff
@@ -197,6 +198,7 @@ public function prepareOrderDetails($order)
                 $mainProduct = $this->productRepository->getById($parentId);
                 $options = (array)$item->getProductOptions();
                 $dataItem['option_id'] = $options['simple_sku'];
+                $dataItem['option_sku'] = $options['simple_sku'];
```

Comment:
Soo `$dataItem['option_id']` is the same as `$dataItem['option_sku']`?

Reply:
I'd really like if we have the comment from `addToCart` here also:
`// for legacy reasons - we have been passing the SKU as ID for the child products`

Reply:
done :)